### DISCUSSION TITLE
How do I properly start in DeviceScript?

### DISCUSSION THREAD

**Discussion by AndyT86** [microsoft/devicescript#390] [Q&A]
I can't enter firmware flash mode at all, there is not a thing working, even "Start Simulator" does not work here, and the video is too small and not able to F for fullscreen, I have no way to troubleshoot the IDE environment. I am going with the Visual Studio Code route.

**Comment by pelikhan** [comment]
Could you tell us more about your current environment?

- visual studio code version: 
- operating system:
- what kind of MCU are you trying to connect
- as for the video, we don't support the F keyboard yet. were you able to keyboard tab into the full screen icon? 
<img width="262" alt="image" src="https://github.com/microsoft/devicescript/assets/4175913/dc5627b6-fcf5-4137-b767-b7959c4a50a1">

**Reply by pelikhan** [reply]
https://github.com/microsoft/devicescript/issues/391
